deploy/addons/registry-aliases/node-etc-hosts-update.tmpl

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       initContainers:
         - name: update
-          image: registry.fedoraproject.org/fedora
+          image: ubuntu:bionic
```

Comment:
why ubuntu ? could it be something smaller ?

Reply:
No reason in particular, that was just my workaround. I've changed it to Alpine 3.11 and fixed the diff command to account for the different between Busybox and GNU.